File: MySolutions/4-a-frequent_symbol/FrequentSymbol.java
import java.io.*;
import java.util.Scanner;
import java.util.StringTokenizer;

public class FrequentSymbol {
    public static void main(String[] arg) {
        FastScanner in = new FastScanner(System.in);
        PrintWriter out = new PrintWriter(System.out);

        String s;
        s = in.next();
        int q;
        q = in.nextInt();
        int[] l = new int[q];
        int[] r = new int[q];
        for (int i = 0; i < q; ++i) {
            l[i] = in.nextInt();
            r[i] = in.nextInt();
        }

        char[] result = new char[q];

        // your code

        for (int i = 0; i < result.length; ++i) {
            out.println(result[i]);
        }

        out.close();
    }

    static class FastScanner {
        BufferedReader br;
        StringTokenizer st;

        FastScanner(InputStream stream) {
            try {
                br = new BufferedReader(new InputStreamReader(stream));
            } catch (Exception e) {
                e.printStackTrace();
            }
        }

        String next() {
            while (st == null || !st.hasMoreTokens()) {
                try {
                    st = new StringTokenizer(br.readLine());
                } catch (IOException e) {
                    e.printStackTrace();
                }
            }
            return st.nextToken();
        }

        int nextInt() {
            return Integer.parseInt(next());
        }
    }
}

File: MySolutions/4-a-frequent_symbol/frequent_symbol.cpp
#include <iostream>
#include <vector>
#include <string>

using std::cin;
using std::cout;
using std::endl;
using std::vector;
using std::string;

int main() {
    string s;
    cin >> s;
    int q;
    cin >> q;
    vector<int> l(q), r(q);
    for (int i = 0; i < q; ++i)
        cin >> l[i] >> r[i];

    vector<char> result(q);

    // your code

    for (int i = 0; i < result.size(); ++i) {
        cout << result[i] << endl;
    }

    return 0;
}